docs/cities/manchester/roles-and-duties.md

```diff
@@ -0,0 +1,111 @@
+---
+id: roles-and-duties
+title: Roles & Duties
+---
+
+Under the main trinity of [Volunteer teams](volunteers/teams.md),namely Outreach, Education & Personal Development,
+generally, we need to act and fulfill different roles in order to run the course smoothly and effectively.
+These roles also help to keep each member being engaged and organised.
+
+A volunteer can wear different hats on the way depending on the schedule and agenda, or maybe special circumstances.
```

Comment:
Suggest a slight reword to 

"A volunteer can wear different hats depending on the schedule, agenda, or in special circumstances."